compass/landice/tests/mismipplus/mismipplus.cfg

```diff
@@ -0,0 +1,13 @@
+[mismipplus]
+
+# the number of cells per core to aim for
+goal_cells_per_core = 300
+
+# the approximate maximum number of cells per core (the test will fail if too
+# few cores are available)
+max_cells_per_core = 5000
+
+# number of cells at 1000m resolution (with no gutter present). This is used
```

Comment:
Based on how I understand this is being used, I think this could be hardcoded and removed from the cfg file.

Reply:
Your totally right. We actually really don't want the user playing with the number, since it would throw off the `ntasks` calculations. 

I've removed the option and comment, and moved both to the [`tasks.py` file](https://github.com/andrewdnolan/compass/blob/4b0f35a01e4606b80d1b86e37910645cfd780e92/compass/landice/tests/mismipplus/tasks.py#L6-L9).